acceptance/bundle/exec/cwd/output.txt

```diff
@@ -0,0 +1,8 @@
+
+>>> cd a/b/c
+
+>>> python3 -c import os; print(os.getcwd())
+[TMPDIR]/a/b/c
+
+>>> [CLI] bundle exec -- python3 -c import os; print(os.getcwd())
+[TMPDIR]
```

Comment:
This looks wrong :)

Reply:
This is by design, the exec command should execute scripts in the bundle root.

Reply:
We now execute from the cwd